tests/server/api/mutations/test_dataset_mutations.py

```diff
@@ -299,6 +302,15 @@ async def test_add_span_to_dataset(
                                             "value": "chain-span-output-value",
                                             "mime_type": "text/plain",
                                         },
+                                        "annotations": {
+                                            "test annotation": {
+                                                "label": "ambiguous",
+                                                "score": 0.5,
+                                                "explanation": "meaningful words",
+                                                "metadata": {},
+                                                "kind": "HUMAN",
```

Comment:
```suggestion
                                                "annotator_kind": "HUMAN",
```
Just to be totally clear